src/pages/TaskList/ui/TaskList.test.tsx

```diff
@@ -10,19 +10,26 @@ import { componentRender } from 'shared/lib/tests';
 import { AnyAction, ThunkDispatch } from '@reduxjs/toolkit';
 import * as dateAndTime from 'shared/lib/helpers/getDateAndTime/getDateAndTime';
 import { getDateAndTime } from 'shared/lib/helpers';
+import { StateSchema } from 'app/store';
+import { testUser } from 'shared/test/TestUser';
 import { TaskList } from './TaskList';
 
 const middlewares = [thunk.withExtraArgument({ firestore })];
 const mockStore = configureStore(middlewares);
 
 describe('TaskList', () => {
     it('renders the task list correctly', async () => {
-        const initialState = {
+        const initialState: StateSchema = {
             tasks: {
                 data: [testTask],
                 isLoading: false,
                 error: undefined,
             },
+            user: {
+                data: testUser,
```

Comment:
could the test for non-logged user be added as well?

Reply:
Added a test